### ISSUE TITLE
TBGRAColor in object inspector 

### ISSUE THREAD

**Issue by lainz** [bgrabitmap/bgrabitmap#227]
It will be nice to have a string property that displays the color like TColor does but with transparency.

Having its own color picker.

**Comment by circular17** [comment]
Indeed. I suppose that would be part of BGRAControls, to be used with the controls.

It would involve a color picker with all components.
